earthfile2llb/converter.go

```diff
@@ -146,6 +146,13 @@ func (c *Converter) fromTarget(ctx context.Context, targetName string, buildArgs
 	for dirKey, dirValue := range relevantDepState.LocalDirs {
 		c.mts.FinalStates.LocalDirs[dirKey] = dirValue
 	}
+
+	for _, kv := range saveImage.Image.Config.Env {
+		fmt.Printf("adding %v\n", kv)
+		k, v := variables.ParseKeyValue(kv)
+		c.varCollection.AddActive(k, variables.NewConstantEnvVar(v), true)
+	}
+
```

Comment:
Nit: Remove whitespace line to help with logical grouping?

Reply:
sorry, I jumped the gun on opening this PR, I accidentally added in some printfs. let me clean this up.